### PR TITLE
Feat/client details

### PR DESCRIPTION
**Pull request by katerinamartysko**
## Description ##

- https://trello.com/c/WrzIQOx5 
- https://trello.com/c/nL8U9tlm

### CODE REVIEW

**Comment by katerinamartysko** [comment]
[Client details](https://trello.com/c/WrzIQOx5/9-client-details)

File: src/components/ClientId.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          <Typography>
+            Address: Country:
+            {client.address.country} <br /> City: {client.address.city} <br /> Street: {client.address.street}
+            <br /> Zip code: {client.address.zipCode}
+          </Typography>
+        </div>
+      </div>
+      {clientIdError && <h1 className="error">Произошла ошибка {clientIdError}</h1>}
+      {isClientIdLoading && (
+        <Box sx={{ display: 'flex' }}>
```
Comment (OleksiiLukovkaIntego) [inline src/components/ClientId.tsx:65]: Remove `{{ display: 'flex' }}` from the component. Please use styles on the `makeStyles`

File: src/components/ClientsList.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  const clients = useSelector((state: State) => state.clients.clientsList);
+
+  const [fetchClients, isClientsLoading, clientError] = useFetching(async () => {
+    const response = await PostService.getClients();
+    dispatch(getClientsList(response.data));
+  });
+
+  useEffect(() => {
+    fetchClients();
+  }, []);
```
Comment (OleksiiLukovkaIntego) [inline src/components/ClientsList.tsx:55]: Please add an empty line after `useEffect`

File: src/components/ClientsList.tsx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+                <Divider className={classes.divider} />
+              </div>
+            ))}
+          </List>
+        </Breadcrumbs>
+      </div>
+      {clientError && <h1 className="error">Произошла ошибка {clientError}</h1>}
+
+      {isClientsLoading && (
+        <Box sx={{ display: 'flex' }}>
```
Comment (OleksiiLukovkaIntego) [inline src/components/ClientsList.tsx:87]: Please remove styles from the component

File: src/utils/them.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    main: '#000000',
+    light: '#ffffff',
+    dark: '#000000',
+    contrastText: '#ff0000',
+  },
+};
+
+export const theme = createTheme({
+  palette,
+  components: {},
```
Comment (OleksiiLukovkaIntego) [inline src/utils/them.ts:14]: Remove empty components

**Review by LexaLukovka — APPROVED** [review]

**Pull request merged**